src/pages/deprecation-policy.md

```diff
@@ -0,0 +1,53 @@
+# Snowplow Software Deprecation Policy
+
+To ensure that we can innovate and evolve our core product, Snowplow reserves the right to deprecate features, components or services it provides to its customers. There can be various reasons for doing so, including but not limited to the following:
+
+- A feature is used by a very small minority of the customers (or not used at all), thus not justifying continued support and investment
+- A more modern, efficient or cost-effective way to achieve the same goal is available, which would benefit all current customers
+- External upstream dependencies of a component (in its current form) are no longer maintained and/or pose a security risk
+
+This document outlines the guarantees we provide and the procedures we follow for deprecation.
+
+## Guarantees
+
+### Notification
+
+Snowplow will notify all customers, as well as the wider community (in case of source-available features or components), of any product or service deprecation 6 months in advance. From that point onwards, we will not make changes to the feature, component or service in question, aside from critical security fixes. After 6 months, it will no longer be supported and will be switched off (where applicable).
```

Comment:
```suggestion
Snowplow will notify all customers, as well as the wider community (in case of source-available features or components), of any product or service deprecation six months in advance. From that point onwards, we will not make changes to the feature, component or service in question, aside from critical security fixes. After six months, it will no longer be supported and will be switched off (where applicable).
```

Reply:
single digit numbers should be spelled out as words